config.js

```diff
@@ -16,12 +16,20 @@ const ConfigOptions = () => {
     let config = {
         isProduction,
         isDev,
-
         webRootDir: '.',
         assetSrcDir: 'src',
         assetDistDir: 'dist',
         applyRevision: true,
 
+        /**
+         * Fozzie-related variables
+         */
+        fozzieSettings: {
+            useFozziePackageVersion: false,
```

Comment:
Also, may be better to make these config settings part of the CSS config object (and the JS config so that they can be set separately).